src/extension.ts

```diff
@@ -136,31 +180,38 @@ class BotoProvider implements vscode.WebviewViewProvider {
 				</div>
 			</body>
 			</html>`;
-	}
+  }
 }
 
-async function pollForEnvironmentLogs(env: any, logChannels: any) {
+async function pollForEnvironmentLogs(
+  env: Environment,
+  logChannels: Record<string, LogChannel>
+) {
   const logPoller = setInterval(async () => {
     const apiKeyCredentials = getApiKeyCredentials();
 
-    const options = {
```

Comment:
not sure why "options" was used. I modified it to axios.get (hopefully I didn't miss anything).

Reply:
tested! Found a bug (not introduced by this PR) and fixed it.
https://github.com/env0/vscode-env0/issues/31